playbook/app/pb_kits/playbook/utilities/_shadow.scss

```
@@ -0,0 +1,23 @@
+@import "../tokens/exports/colors";
+@import "../tokens/exports/opacity";
+
+$shadow_none: 0 0 0 0 transparent;
+$shadow_deep: 0 4px 10px 0 rgba($shadow, 0.16);
+$shadow_deeper: 0 12px 28px 0 rgba($shadow, 0.18);
+$shadow_deepest: 0 30px 38px 4px $shadow, 0 2px 14px 4px rgba($shadow, 0.1);
+
+.shadow_none {
+  box-shadow: $shadow_none;
+}
+
+.shadow_deep {
+  box-shadow: $shadow_deep;
+}
+
+.shadow_deeper { 
+  box-shadow: $shadow_deeper;
+}
+
+.shadow_deepest { 
+  box-shadow: $shadow_deepest; 
+}
```

```suggestion
@mixin shadow-classes($shadows-list) {
  @each $name, $shadow in $shadows-list {
    .shadow_#{$name} {
      box-shadow: $shadow;
    }
  }
}

@include shadow-classes($box_shadows);
```

Let's make this smarter. We already have an awesome map we can loop over to dynamically generate these.